pytrafikverket/__main__.py

```diff
@@ -73,7 +74,7 @@ async def async_main(loop):
                 time = datetime.strptime(args.date_time, DATE_TIME_INPUT)
 
                 train_stop = await train_api.async_get_train_stop(
-                    from_station, to_station, time
+                    from_station, to_station, time, product_description=args.description
```

Comment:
should it not be `product_description=args.product_description`?
And as it's only for trains could we maybe rename it to `train_product` or something?

Reply:
Oops, good point!